### DISCUSSION TITLE
Optimal of Quadratic Optimization

### DISCUSSION THREAD

**Discussion by 0xF0F0F0F** [yalmip/YALMIP#1103] [General discussion] [answered]
Considering a quadratic problem (QP):

min_y  y'*H*y + f'*y
s.t. 0<= (y/X) <= ub
     sum(y) = X

where H is a positive definite matrix; X>0; sum(ub)>1; y is a (nx1) decision variable and let y* be the optimal of the quadratic problem.

By using different values of X in the QP and keep all other parameters constant, can anyone explain to me, why the ratio y*/X is always constant regardless the value of X?

**Comment by johanlofberg** [comment, marked as answer]
That is simply not true generically

```
H = randn(3);H=H*H';
f = randn(3,1);
U = 1 + rand(3,1);

y = sdpvar(3,1);
for i = 1:10
    X = rand(1);
    optimize([0<=y/X<=U,sum(y)==X],y'*H*y + f'*y);
    value(y/X)
end
```

Having said that, this is the wrong forum for general optimization questions
